### PR TITLE
feat: update to Bot API 6.2

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Also updates deps, docs, and rearranges imports based on Deno's new feature

### CODE REVIEW

**Comment by roj1512** [comment]
What is Deno's new feature?

**Comment by KnorpelSenf** [comment]
I have no idea when it was introduced, but Deno can now organise imports automatically through the LSP, even though https://github.com/denoland/deno/issues/12672 is still pending.

**Review by EdJoPaTo — APPROVED** [review]
Diff seems reasonable, still works in two of my bots so it does not seem to break anything (which I would assume based on the diff).

**Review by deptyped — APPROVED** [review]

File: src/context.ts [hunk 1]
```diff
@@ -544,7 +545,7 @@ export class Context implements RenamedUpdate {
     }
 
     /**
-     * Context-aware alias for `api.copyMessage`. Use this method to copy messages of any kind. Service messages and invoice messages can't be copied. The method is analogous to the method forwardMessage, but the copied message doesn't have a link to the original message. Returns the MessageId of the sent message on success.
+     * Context-aware alias for `api.copyMessage`. Use this method to copy messages of any kind. Service messages and invoice messages can't be copied. A quiz poll can be copied only if the value of the field correct_option_id is known to the bot. The method is analogous to the method forwardMessage, but the copied message doesn't have a link to the original message. Returns the MessageId of the sent message on success.
```
Comment (roj1512) [inline src/context.ts:548]: Do we need back tick for `correct_option_id`?

Comment (KnorpelSenf) [reply src/context.ts:548]: No. As stated in point one [here](https://github.com/grammyjs/types#differences-to-the-bot-api), we don't manually add formatting to the parts of the docs that are copied from the website. (At least not yet.)

This is only done for manually written docs, such as the prefixing sentence at the beginning of this line.

**Review by Loskir — APPROVED** [review]

**Pull request merged**